content/docs/04-clusters/4-nested-clusters/nested-cluster-quickstart.md

```diff
@@ -15,113 +15,134 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Getting Started with Nested Clusters
 
-With the Palette Nested Cluster option, you can deploy additional Kubernetes clusters within an existing cluster to provide teams with sandboxed environments where you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.
+With the Palette Nested Cluster option, you can deploy additional Kubernetes clusters, within an existing cluster, to provide teams with nested cluster environments. There are certain advantages to use a nested cluster environment. You are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.
 
-To access a Palette Sandbox, go to [SpectroCloud](https://spectrocloud.com) <!--confirm link--> and use your Github or Google SSO. You will be prompted to create a Palette account.
-You are allowed to deploy a managed Nested Cluster. See below under [Deploying a Nested Cluster](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster), follow the steps below under 
+Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a managed Nested Cluster from within the Palette orchestrator.
 
+## Prerequisites
 
-# Prerequisites
+- You will need to have access to a Palette Account. Refer to the [Getting Started with Palette](/getting-started) page.
 
-- You will need to have access to a Palette Account. Refer to the [Getting Started with Palette](/getting-started).
 
+- Ensure you have a functioning [cluster](/clusters/new-clusters) already configured. During the cluster creation, you will be able to configure it as a Host Cluster.
 
-- Ensure you have a functioning cluster already configure. Refer to the [New Clusters](/clusters/new-clusters) page for more information.
+  **Hosted Clusters**
 
+  There are three ways to engage with a Nested Cluster on Palette.
 
-# Deploying a Nested Cluster
-1. From the slide menu, select the **Clusters** tab and click the **Nested Clusters** tab to list the available Nested Clusters, and then select **Add New Nested Cluster**.
+   1. When you [create](/clusters/new-clusters) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a new Host Cluster, there is an option to **Enable Nested Clusters**. Refer to the [Cluster Endpoint](/clusters/nested-clusters/cluster-quickstart#loadbalancer) step below for more information on how to configure this option. The same settings listed below are available, when you are enabling a Nested Cluster and when deploying a Host Cluster deployment wizard.
```

Comment:
"below are available, when you are enabling" - no comma needed. Should be "below are available when you are enabling"